landing-page/LandingPage.js

```diff
@@ -0,0 +1,52 @@
+import { gql, useQuery } from "@apollo/client";
+import { Box } from "@react-native-material/core";
+
+import SHButton from "../components/SHButton";
+import LoadingPage from "../loading-page/LoadingPage";
+
+const SERVICES_QUERY = gql`
+  {
+    services {
+      edges {
+        node {
+          id
+          name
+          identifier
+          active
+        }
+      }
+    }
+  }
+`;
+
+const servicesList = [];
+
+export default function LandingPage({ navigation }) {
+  const { data, loading } = useQuery(SERVICES_QUERY);
+  if (loading) {
+    return <LoadingPage />;
+  }
+  for (let i = 0; i < data.services.edges.length; i++) {
+    let localIcon = "";
+    switch (data.services.edges[i].node.name) {
+      case "Katzenprofil":
+        localIcon = "cat";
+        break;
+      case "Katzenfütterung":
+        localIcon = "food";
+        break;
+      case "Katzenerkennung":
+        localIcon = "smoke-detector-variant";
+        break;
+    }
+    servicesList.push(
```

Comment:
Das rendern der Icons würde ich eher in der SHButton Component sehen, bzw vielleicht sogar in einer eigenen Component? Um den code aufgeräumter zu haben. Vielleicht haben wir auch eine Möglichkeit das laden der icons generischer zu geestalten? Was passiert wenn der Service nicht "Katzenprofil", "Katzenfütterung" oder "Katzenerkennung" heißt?